tasks/Task.cpp

```diff
@@ -906,6 +906,12 @@ void Task::acquisitionConfiguration(Arena::IDevice& device)
         ptp_sync_frame_rate->SetValue(config.frame_rate);
     }
     else {
+        double frame_timeout = _image_config.get().frame_timeout.toSeconds();
+        double seconds_per_frame = 1 / config.frame_rate;
+        if (frame_timeout < seconds_per_frame) {
```

Comment:
```suggestion
        double time_length_per_frame = 1 / config.frame_rate;
        if (frame_timeout < time_length_per_frame) {
```

Reply:
That's nitpicking ... the time is in seconds.